cases/templates/cases/case_detail_staff.html

```diff
@@ -313,7 +313,25 @@ <h2 class="lbh-heading-h2 section-heading">Timeline</h2>
                 {{ entry.action.notes }}
             </div>
         {% endif %}
-
+        {% if entry.action.files %}
+            {% for file in entry.action.files.all %}
+                <p>
+                    <a class="nw-link--no-visited-state lbh-body-s" href="{% url "action-file" case.id entry.action.id file.id %}">
+                        {{ file.original_name }} ({{file.human_readable_size}})
+                    </a>
+                </p>
+            {% endfor %}
+            <div class="nw-printable-timeline-details" hidden>
```

Comment:
I do not think this is needed - the other `<details>` have it because they do not print if closed. No such issue here.